tests/e2e/tests/admin/api-tokens.spec.ts

```diff
@@ -16,16 +15,18 @@ const createAPIToken = async (page, tokenName, duration, type) => {
   await page.getByRole('option', { name: type }).click();
 
   await page.getByRole('button', { name: 'Save' }).click();
-
   await expect(page.getByText('Make sure to copy this token')).toBeVisible();
   await expect(page.getByText('Expiration date:')).toBeVisible();
 };
 
 test.describe('API Tokens', () => {
   test.beforeEach(async ({ page }) => {
-    await resetDatabaseAndImportDataFromPath('with-admin.tar');
-    await page.goto('/admin');
-    await login({ page });
+    await sharedSetup('ctb-edit-st', page, {
```

Comment:
This is just to reduce flakiness since this test kept failing about 20% of the time